ingestion/src/metadata/data_insight/processor/web_analytic_report_data_processor.py

```diff
@@ -239,7 +239,7 @@ def _get_user_details(self, user_id: str) -> Optional[dict]:
         )
 
         if not user_entity:
-            return None
+            return {}
```

Comment:
is this really needed? can be more error prone than reviewing a null value cc @TeddyCr

Reply:
yes, pass a dict to the user id key we get from the event data and retrieve those details to process it down below.